chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -362,9 +485,9 @@ public DeletedUser deleteUser(@PathVariable("userId") Long userId) throws UserRe
     @PreAuthorize("isAuthenticated()")
     public @ResponseBody UsersResponse getUsers() {
         List<User> users = null;
-        if (AuthUtil.getCurrentUser().getAuthenticationSystem().equals(AuthenticationSystem.COGNITO)) {
+        if (ff4j.check(FeatureList.SSO)) {
```

Comment:
This endpoint is marked as Deprecated/DeprecatedApi but I am not sure that it is. It is being used from the Users -> User Management page in both flag states. If it is truly deprecated then I would expect a different endpoint to be called when the SSO flag is ON.